src/script/lua_api/l_object.cpp

```diff
@@ -1174,6 +1174,47 @@ int ObjectRef::l_set_look_yaw(lua_State *L)
 	return 1;
 }
 
+// clear_fov(self)
+int ObjectRef::l_clear_fov(lua_State *L)
+{
+	NO_MAP_LOCK_REQUIRED;
+	ObjectRef *ref = checkobject(L, 1);
+	RemotePlayer *player = getplayer(ref);
+	if (player == NULL) return 0;
+
+	getServer(L)->setPlayerFov(player, 0.0f);
+
+	return 1;
+}
+
+// set_fov(self, degrees)
+int ObjectRef::l_set_fov(lua_State *L)
+{
+	NO_MAP_LOCK_REQUIRED;
+	ObjectRef *ref = checkobject(L, 1);
+	RemotePlayer *player = getplayer(ref);
+	if (player == NULL) return 0;
+
+	f32 fov = luaL_checknumber(L, 2);
+	getServer(L)->setPlayerFov(player, fov);
+
+	return 1;
```

Comment:
Does not return anything. Return 0.

Reply:
I wasn't sure about what the return value was supposed to be. Fixed, thanks